ansible/roles/schulcloud-server-core/templates/admin-api-server-svc.yml.j2

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: admin-api-serv-svc
```

The name shoud be **api-admin-svc**
```suggestion
  name: api-admin-svc
```

---

done